### PR TITLE
Rebuild example, update iOS dependency to 6.0.x

### PR DESCRIPTION
**Pull request by frankus**
Update with files from refreshed project, including ApptentiveKit iOS SDK (v6).

### CODE REVIEW

**Review by HarryAWoodworth — APPROVED** [review]

File: android/build.gradle [hunk 1]
```diff
@@ -1,36 +1,132 @@
...
-    }
+  // Buildscript is evaluated before everything else so we can't use getExtOrDefault
+  def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['Apptentive_kotlinVersion']
 
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
-    }
+  repositories {
+    google()
+    mavenCentral()
```
Comment (PoornimaApptentive) [inline android/build.gradle:7]: jcenter() can be removed.

Comment (PoornimaApptentive) [reply android/build.gradle:7]: There are a few outdated code that need to be cleaned up from `build.gradle`. If releasing is the priority that can be done when updating android new SDK.

File: android/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Apptentive_kotlinVersion=1.3.50
```
Comment (PoornimaApptentive) [inline android/gradle.properties:1]: Here as well.

File: apptentive-react-native.podspec [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
   s.name         = "apptentive-react-native"
-  s.version      = "5.8.3"
-  s.summary      = "Apptentive SDK module for React Native"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => "11.0" }
```
Comment (PoornimaApptentive) [inline apptentive-react-native.podspec:13]: Getting 404 for this repo.

Comment (PoornimaApptentive) [reply apptentive-react-native.podspec:13]: Also, we should move this to the Apptentive git repository, if it is staying in Harry's private space.

**Comment by PoornimaApptentive** [comment]
Overall looks good to me.

File: apptentive-react-native.podspec (hunk 3 above)
Comment (frankus) [reply apptentive-react-native.podspec:13]: Good catch!

**Review by PoornimaApptentive — APPROVED** [review]

**Pull request merged**